demos/pedestrian_tracker_demo/cpp/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM wahtari/openvino_openmodelzoo_demos:runtime
+
+RUN mkdir /models && cd /models && \
+    wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/3/pedestrian-detection-adas-0002/FP32/pedestrian-detection-adas-0002.xml && \
+    wget https://storage.openvinotoolkit.org/repositories/open_model_zoo/2022.1/models_bin/3/pedestrian-detection-adas-0002/FP32/pedestrian-detection-adas-0002.bin
+
+COPY pedestrian_tracker_demo /demo
+
+CMD ["/demo", "-no-show", "-at", "openpose", "-i", "/models/pedestrian-detection-adas-0002.bin", "-m", "/models/pedestrian-detection-adas-0002.xml"]
```

Comment:
Check the arguments of the pedestrain demo.  Looking at the README file:

```
./pedestrian_tracker_demo -i <path_video_file> \
                          -m_det <path_to_model>/person-detection-retail-0013.xml \
                          -m_reid <path_to_model>/person-reidentification-retail-0277.xml \
                          -d_det GPU
                          -at ssd
```


`-i`, `-m_det`, `-m_reid`, and `-at` are required to start this app. Looking at the models.lst, it seams so, that this demo requires two AI models.

- person-reidentification-retail-???? for the `-m_reid` flag
- ex. person-detection-retail-???? for the `-m_det` flag
- `-at ssd` for the at flag, if we use the upper model as explained in the models.lst file
- the `-i`flag can be anything, because we use our own camera implementation.

Download both models via wget in the upper section of the Dockerfile. Start the app with:

```
CMD ["/demo", "-no-show", "-i", "alvium", "-at", "ssd", "-m_det", "/models/person-detection-retail-????.xml", "-m_reid", "/models/person-reidentification-retail-????.xml"]
```